lib/kernel/doc/kernel_app.md

```diff
@@ -130,6 +130,26 @@ For more information about configuration parameters, see file
     node. If a node goes down, it must thereafter be explicitly connected. See
     `m:net_kernel`.
 
+- **`epmd_module = module()`{: #epmd_module }** - Configures the module
+  responsible for communication with [epmd](epmd_cmd.md). If this parameter
+  is undefined, it defaults to `erl_epmd`.
+
+  The now deprecated command line argument
+  [`-epmd_module <module>`](`e:erts:erl_cmd.md#epmd_module`) has the same
+  effect as the `epmd_module` configuration parameter. If this configuration
+  parameter is defined, it will override the command line argument.
+
+- **`erl_epmd_listen_port = integer()`** - Configures the port used by `m:erl_epmd`
```

Comment:
Perhaps we don't need the erl prefix?

Reply:
That was my initial thought, but my reasoning is that this config option is only applicable when `epmd_module = erl_epmd`, in other words specific to `erl_epmd`. Does that makes sense?